index.html

```diff
@@ -0,0 +1,204 @@
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <title>Wasyihun-portfolio</title>
+  </head>
+  <body>
+    <header class="navbar">
+      <div>
+        <a class="My_logo">MyLogo</a>
+      </div>
+      <nav class="menu_icon">
+        <div class="menu_icon"></div>
+        <div class="menu_icon"></div>
+        <div class="menu_icon"></div>
+      </nav>
+    </header>
+    <section class="Body_content">
+      <div class="Body_content_title">
+        <h1>
+          I’m Wasyihun<br />
+          Glad to see<br />
+          you!
+        </h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <h2>LET’S CONNECT</h2>
+        <ul>
+          <li><img src="image/Twittwer_vector.png" alt="" /></li>
+          <li><img src="image/Linkedin_icon.png" alt="" /></li>
+          <li><img src="image/M_vector.png" alt="" /></li>
+          <li><img src="image/Github_vector.png" alt="" /></li>
+          <li><img src="image/Pieace_vector.png" alt="" /></li>
+        </ul>
+      </div>
+    </section>
+    <section class="Works">
+      <main class="Works_main">
+        <img class="big_pictures" src="image/Picture_1.png" alt="" />
+        <h2>Tonic</h2>
+        <ul class="Works_Tony">
+          <li>CANOPY</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>Back End Dev</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </main>
+      <main class="Works_main">
+        <img class="big_pictures" src="image/picture_2.png" alt="" />
+        <h2>
+          Multi-Post<br />
+          Stories
+        </h2>
+        <ul class="Works_Tony">
+          <li>CANOPY</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>Back End Dev</li>
+          <li><img src="image/Card_1.png" alt="" /></li>
+          <li>2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="languages">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </main>
+      <main class="Works_main">
+        <img class="big_pictures" src="image/picture_3.png" alt="" />
```

Comment:
The image link is broken.
![image](https://user-images.githubusercontent.com/61361037/219758245-47d464d5-71ba-4e45-aa22-33a7db619c64.png)